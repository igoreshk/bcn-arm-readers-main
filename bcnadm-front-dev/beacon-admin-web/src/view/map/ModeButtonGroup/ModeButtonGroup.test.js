import React from 'react';
import { shallow } from 'enzyme';

import { ModeButtonGroup } from './index';

const props = {
  scaleDistance: 10,
  translate: jest.fn(),
  match: {
    params: {
      layer: ''
    }
  }
};

const wrapper = shallow(<ModeButtonGroup {...props} />);

describe('mock', () => {
  it('render component', () => expect(wrapper.length).toEqual(1));
  it('snapshot for ModeButtonGroup', () => expect(wrapper).toMatchSnapshot());
});
