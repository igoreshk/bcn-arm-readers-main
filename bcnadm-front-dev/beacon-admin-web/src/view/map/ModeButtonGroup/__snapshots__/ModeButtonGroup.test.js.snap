// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`mock snapshot for ModeButtonGroup 1`] = `
<div
  className="modeButtonGroup"
>
  <div
    className="firstModeButtonGroup"
  >
    <withRouter(ModeButton)
      active={false}
      buttonId="areas"
      buttonType="MODE"
      disabled={true}
      icon={
        <Memo
          className="inactiveButtonIcon"
        />
      }
      link="areas"
    />
    <withRouter(ModeButton)
      active={false}
      buttonId="routing"
      buttonType="MODE"
      disabled={false}
      icon={
        <Memo
          className="inactiveButtonIcon"
        />
      }
      link="routing"
    />
    <withRouter(ModeButton)
      active={false}
      buttonId="beacons"
      buttonType="MODE"
      disabled={false}
      icon={
        <MdWifiTethering
          className="inactiveButtonIcon"
        />
      }
      link="beacons"
    />
    <withRouter(ModeButton)
      active={false}
      buttonId="readers"
      buttonType="MODE"
      disabled={false}
      icon={
        <MdWifiTethering
          className="inactiveButtonIcon"
        />
      }
      link="readers"
    />
  </div>
  <div
    className="secondModeButtonGroup"
  >
    <withRouter(ModeButton)
      active={false}
      buttonId="scaling"
      buttonType="MODE"
      icon={
        <Memo
          className="inactiveButtonIcon"
        />
      }
      link="scaling"
    />
    <withRouter(ModeButton)
      active={false}
      buttonId="monitoring"
      buttonType="MODE"
      disabled={false}
      icon={
        <Memo
          className="inactiveButtonIcon"
        />
      }
      link="monitoring"
    />
    <withRouter(ModeButton)
      active={false}
      buttonId="drawing"
      buttonType="MODE"
      disabled={false}
      icon={
        <Memo
          className="inactiveButtonIcon"
        />
      }
      link="drawing"
    />
  </div>
</div>
`;
